section3/reflection.md

```diff
@@ -2,12 +2,23 @@
 
 1. In Section 3, we removed even more supports and pushed you to learn a bit more independently, read through technical documentation, and use your resources more. How did that feel?
 
+This section was much more difficult than the previous sections. It was frustrating at times, but we have great peers in this class that are more than happy to help out when you can't find the answer on your own.
+
 2. Think of a web or mobile application you use regularly. Where is an example of a place that the developers probably use Arrays? Explain.
 
+eBay would probably use an array when it's showing search results, using the terms you entered to compare to what others have listed in item descriptions.
+
 3. In your own words, explain what an Array method is.
 
+An array method is a way to modify part of an array, while not affecting all the elements of it.
+
 4. What is the purpose of a loop? When we would use a loop in conjunction with array vs. without?
 
+A loop is used to have the program perform a function multiple times, such as a countdown timer, or listing birthdays, etc. A loop would be used with an array when we want to make changes to multiple pieces of data multiple times.
+
 5. If you had to teach a 5 year old about loops, how would you explain it?
 
+A loop is repeating the same steps multiple times, and getting different results.
+
 6. What questions do you still have about Arrays and loops?
+In the last question on the loops-arrays.js file, I'm still confused why we don't have a condition for the else. When I try to run it with the condition of >= 18, it would not give the correct console log to the variables, it made them all the same. 
```

Comment:
Well `else` alone does not accept a condition; it is truly for `anything else`. Without seeing the exact code you wrote though, I don't know if I can provide much more info?